### ISSUE TITLE
Experiment with stratified sampling on Research Chapter 4

### ISSUE THREAD

**Issue by zxul767** [zxul767/fastai-annotated#3]
In research notebook for chapter 4, an ensemble of binary classifiers (for the MNIST classification problem) is created which achieves ~90% accuracy on the validation set.

However, the strategy used to train the individual classifiers uses a general random sampling, which makes it perfectly legitimate that some of the samples will contain more classes of a certain digit.

Would the validation accuracy rise if we sampled the corresponding proportion from each digit class instead?
